src/app/file/bmp_format.cpp

```diff
@@ -82,13 +84,20 @@ FileFormat* CreateBmpFormat()
   return new BmpFormat;
 }
 
-#define BI_RGB          0
-#define BI_RLE8         1
-#define BI_RLE4         2
-#define BI_BITFIELDS    3
+#define BI_RGB            0
+#define BI_RLE8           1
+#define BI_RLE4           2
+#define BI_BITFIELDS      3
+#define BI_ALPHABITFIELDS 6
 
-#define OS2INFOHEADERSIZE  12
-#define WININFOHEADERSIZE  40
+#define OS2FILEHEADERSIZE   14
+
+#define OS2INFOHEADERSIZE   12
+#define WININFOHEADERSIZE   40
+#define BV2INFOHEADERSIZE   52
+#define BV3INFOHEADERSIZE   56
+#define BV4INFOHEADERSIZE  108
+#define BV5INFOHEADERSIZE  124
 
 typedef struct BITMAPFILEHEADER
```

Comment:
This code is quite old (it's from C code), we can replace all `typedef struct NAME { ... } NAME;` with just `struct NAME { ... };` (which is the more C++-like syntax we are using now). And we can define all new structs with this syntax.